modules/cachedb_local/cachedb_local.c

```diff
@@ -69,6 +75,8 @@ static param_export_t params[]={
 	{ "exec_threshold",     INT_PARAM, &local_exec_threshold },
 	{ "cache_collections",  STR_PARAM|USE_FUNC_PARAM, (void *)parse_collections },
 	{ "cachedb_url",        STR_PARAM|USE_FUNC_PARAM, (void *)store_urls },
+	{ "replication_cluster",INT_PARAM, &replication_cluster },
+	{ "restart_persistency",STR_PARAM, &restart_persist },
```

Comment:
maybe a better naming would be `cluster_id` and `cluster_persistency` - these 2 params are 100% related to clustering support.

Reply:
I changed this. I noticed there is no consistent way for the naming of cluster variables across the different modules - maybe this should be targeted at some point.

Reply:
yes, you are perfectly right - we are also trying to 'standardize' the name of the params, so it a kind of work in progress :)